process/smartContract/hooks/blockChainHook.go

```diff
@@ -480,6 +494,42 @@ func (bh *BlockChainHookImpl) SetCurrentHeader(hdr data.HeaderHandler) {
 	bh.mutCurrentHdr.Unlock()
 }
 
+// SaveCompiledCode saves to cache and storage the compiled code
```

Comment:
maybe rephrase to `// SaveCompiledCode saves the compiled code to cache and storage`

Reply:
done